### PR TITLE
[FIX] [10.0] account_financial_report_qweb

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
This PR solve error generated when trial balance report is generated with 'Show partner details' selected.
image

### CODE REVIEW

File: account_financial_report_qweb/report/trial_balance.py [hunk 1]
```diff
@@ -230,7 +231,8 @@ def _inject_partner_values(self):
     initial_balance,
     debit,
     credit,
-    final_balance
+    final_balance,
+    currency_name
```
Comment (pedrobaeza) [inline account_financial_report_qweb/report/trial_balance.py:235]: for not having 2 lines diff, insert currency_name before.

File: account_financial_report_qweb/report/trial_balance.py [hunk 2]
```diff
@@ -241,7 +243,8 @@ def _inject_partner_values(self):
     rpg.initial_balance AS initial_balance,
     rpg.final_debit - rpg.initial_debit AS debit,
     rpg.final_credit - rpg.initial_credit AS credit,
-    rpg.final_balance AS final_balance
+    rpg.final_balance AS final_balance,
+    rpg.currency_name AS currency_name
```
Comment (pedrobaeza) [inline account_financial_report_qweb/report/trial_balance.py:247]: The same about 2 lines

File: account_financial_report_qweb/report/trial_balance.py [hunk 3]
```diff
@@ -257,4 +260,4 @@ def _inject_partner_values(self):
             self.general_ledger_id.id,
             self.id,
         )
-        self.env.cr.execute(query_inject_partner, query_inject_partner_params)
+        self.env.cr.execute(query_inject_partner, query_inject_partner_params)
```
Comment (pedrobaeza) [inline account_financial_report_qweb/report/trial_balance.py]: Don't remove final line

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by pedrobaeza** [comment]
Check Travis

**Comment by JordiBForgeFlow** [comment]
Ping @hveficent

**Review by feketemihai — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
@hveficent Can you rebase and review the conflicts?

**Comment by HviorForgeFlow** [comment]
@jbeficent ready!

**Pull request merged**